pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["setuptools >= 61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "cockpit"
+version = "0"
+authors = [{name = "Allison Karlitskaya", email = "allison.karlitskaya@redhat.com"}]
+readme = "README.md"
+license = {file = "COPYING"}
+classifiers = ["License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)"]
+
+[project.urls]
+homepage = "https://cockpit-project.org/"
+repository = "https://github.com/cockpit-project/cockpit/"
+
+[tool.setuptools.packages.find]
+where=["src"]
+include=["cockpit"]
+
+[tool.mypy]
+mypy_path = 'src'
+
+[tool.pylint]
+max-line-length = 118
```

Comment:
I'd vote for just entirely disabling line length checking in pylint. We already do that in flake8, and then we don't need to keep the number in sync.